### ISSUE TITLE
The actual image not copied to CopyQ history when "Copy Image" from browser

### ISSUE THREAD

**Issue by latipun7** [hluk/CopyQ#2128]
## Describe the bug
Image not copied to CopyQ history when "Copy Image" from browser. Image plugin also enabled. I'm able to see image copied to clipboard from flameshot for example. But not from browser "Copy Image". 
![image](https://user-images.githubusercontent.com/20012970/193294418-e01cc032-913b-4c28-8784-1104510cac62.png)

When I click "Copy Image" in browser, my clipboard really did has image, so I can paste here in github comment for example or paste to image editor. But if I copy from copyQ, when I paste here, it only the link/plain text.
Here's my clipboard in CopyQ looks like:
![image](https://user-images.githubusercontent.com/20012970/193298029-500fe304-8c1c-406c-ad64-3f2e4c71b351.png)

## To Reproduce
- right click image and "Copy image"
- see the content of clipboard using <kbd>Ctrl+Shift+C</kbd>
- the clipboard really showing the image
![image](https://user-images.githubusercontent.com/20012970/193096573-d2fe91a3-32ac-45b8-8054-0462e543d943.png)
- see the content of CopyQ of that image <kbd>F4</kbd>, no image
![image](https://user-images.githubusercontent.com/20012970/193097670-e5c28ebe-59ba-4ed3-b60f-d62cb342a7bb.png)

## Expected behavior
png image is saved in CopyQ history instead of plain text and url

## Version, OS and Environment
CopyQ Clipboard Manager 6.3.2
Qt: 5.15.6
KNotifications: 5.98.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: Arch Linux
Window Manager: AwesomeWM

**Comment by hluk** [comment]
Can try disabling any automatic commands? To do that uncheck any entries in Commands dialog (F6 shortcut) that are tagged with a small clipboard icon.

**Comment by latipun7** [comment]
sorry @hluk , it still not work. even I only has this command, the entries only saved as plain text & url text.
![image](https://user-images.githubusercontent.com/20012970/194433264-2d98ebdf-6b52-4318-90f7-4c9df9a4ced2.png)

**Comment by hluk** [comment]
What does the `text/plain` item format contain (check with F4)? I don't see it in the clipboard screenshots.

**Comment by latipun7** [comment]
These two are the content of <kbd>F4</kbd>:
![image](https://user-images.githubusercontent.com/20012970/194483527-2ea7424d-9860-489e-9359-c99859b3c9ae.png)
![image](https://user-images.githubusercontent.com/20012970/194483588-42b08503-4062-4d04-8f34-15f20ec6b0ff.png)

`text/plain` has some content of `text/uri-list` from clipboard content (<kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>c</kbd>)
![image](https://user-images.githubusercontent.com/20012970/194484188-a2605f3d-ea03-4615-a5fc-26a97d71d044.png)

**Comment by hluk** [comment]
Oh, it looks like Qt on X11 will provide `text/plain` format when `text/uri-list` is copied.

CopyQ ignores image formats in clipboard if there is also `text/plain` format because it causes problems with some spreadsheet apps which provide both text and images but sometimes the images are huge and apps get stuck or even crash.

BTW, I cannot reproduce the behavior with Wayland.

**Comment by latipun7** [comment]
Oh dem. Any workarounds?

**Comment by hluk** [comment]
Adding the following command should help ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them)):

```ini
[Command]
Automatic=true
Command="
    copyq:
    const imageFormat = 'image/png';
    const imageData = clipboard(imageFormat);
    if (imageData.length > 0) {
        setData(imageFormat, imageData);
    }"
Icon=\xf1c5
Name=Save Images
```

**Comment by latipun7** [comment]
Yeah it did. Thank you for your support ✨

**Comment by eggbean** [comment]
I've just started using CopyQ and I seem to have the same problem going on, on Windows. When copying from Chrome, the image wasn't being displayed in the main window, but the data could be seen in the F4 window (eg. as image/png).

I added the above automatic command and now they do display in the window, but if I try to Ctrl-c copy from the clipboard tab and Ctrl-v an application, I just get an icon (in the case of Twitter, a camera icon).

If I Ctrl-v paste it into another tab I just get a blank page icon, but if I drag and drop from the clipboard tab to the other tab, it appears as an image which *can* be Ctrl-c copied and pasted into an application successfully.

I impressed with the power of this program and the possibilities of what I can do with it, but I need to fix this basic functionality. I'm wondering if there is something wrong with my installation, as I can't see many other people with this issue.
